MapboxNavigation/RouteController.swift

```diff
@@ -110,9 +107,46 @@ extension RouteController: CLLocationManagerDelegate {
             }
         }
         
+        if routeProgress.currentLegProgress.stepIndex == 0 {
+            let step = routeProgress.currentLegProgress.currentStepProgress.step
+            let stepLocation = CLLocation(latitude: step.coordinates!.first!.latitude, longitude: step.coordinates!.first!.longitude)
```

Comment:
Oh yeah, I guess the first steps maneuver location is the beginning of the route